assets/js/adminCtrl.js

```diff
@@ -4,6 +4,14 @@ define([
 ], function () {
 
     var adminCtrl = function ($scope) {
+
```

Comment:
Maybe, because we have a new page, we could stick this new stuff in a new controller just for that page?

The ideal situation would probably be a directive, but I'm happy with leaving that for just now.